oat-sa/extension-tao-publishing
# Feature/tao 10142/add dialog to publish to remote environments

Jira ticket: https://oat-sa.atlassian.net/browse/TAO-10142

The goal of this development is to add a new action "Publish To Remote" with possibility to publish delivery to selected remote environments any time. 

How to test:
- Add remote environment(s) in "Settings -> Remote Environments"
- Create local delivery (do not check `Sync to remote environments` on new delivery form)
- Open newly created delivery
- New button "Publish To Remote" should appear in actions panel
- Click "Publish To Remote" -> select remote environment(s) -> press "Publish" button
- Check selected remote environments, new delivery must be created (may take some time depending on delivery size)

*Note: At the moment it uses old task (for backward compatibility) and exports QTI test package when task is executed. This functionality will be optimized in future when old "remote publishing" functionality will be deleted.
